tests/ccmcluster.py

```diff
@@ -113,7 +113,10 @@ def parse_cluster_status(self, stdout):
         """
         nodes_status = []
         for line in stdout.split("\n")[2:]:
-            node, status = line.split(":")
+            chunks = line.split(":")
+            if len(chunks) != 2:
```

Comment:
presumably it was an empty which broke the test. an alternative is:
if not line:
    # ignore empty lines
    continue
node, status = line.split(":", 1)
```

Reply:
> presumably it was an empty line which broke the test. an alternative is:

tbh, I have no idea which line broke it, but it is better to make it as resiliant as possible.

> ```python
> if not line:
>     # ignore empty lines
>     continue
> node, status = line.split(":", 1)
> ```

This code will still fail if line does not contain ':'.

Reply:
yes, by looking at the error message, i think the offending line was empty. 

> as resiliant as possible.

i disagree. we program with a contract. "as resilient as possible" without a contract is error-prune, because we don't have a standard or contract for "as resilient as possible".

anyway, this is not that important in this context.